x-pack/solutions/security/plugins/security_solution/public/detections/components/attacks/content.tsx

```diff
@@ -46,6 +55,19 @@ export const AttacksPageContent = React.memo(({ dataView }: AttacksPageContentPr
   const containerElement = useRef<HTMLDivElement | null>(null);
 
   const { globalFullScreen } = useGlobalFullScreen();
+  const [selectedConnectorNames, setSelectedConnectorNames] = useState<string[]>([]);
+  const {
+    services: { settings },
+  } = useKibana();
+
+  const { http, inferenceEnabled } = useAssistantContext();
+  const { data: aiConnectors } = useLoadConnectors({
+    http,
+    inferenceEnabled,
+    settings,
+  });
+  // TODO: Filter names by connectors that generated attacks
+  const aiConnectorNames = useMemo(() => aiConnectors?.map((item) => item.name), [aiConnectors]);
```

Comment:
We need to replace this once we integrate the data fetching logic

Reply:
After discussing it with the team we have decided to fetch the data outside of the table